api-dummy/public/schemas/libero/article/body.rng

```diff
@@ -1,20 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<grammar xmlns="http://relaxng.org/ns/structure/1.0" ns="http://libero.pub">
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
 
     <include href="../core.rng"/>
 
     <start>
+        <ref name="libero.body"/>
+    </start>
 
-        <element name="body">
-            <ref name="libero.body.content"/>
-        </element>
+    <div>
+        <a:documentation>body element</a:documentation>
```

Comment:
Alternatives:

- Comments
- XHTML

See http://books.xmlschemata.org/relaxng/relax-CHP-13-SECT-2.html.

Reply:
XHTML seems too specific, while this kind of tags allow for introspection by tools (whether it is a build or process validating a schema or something that displays the text of the annotations) in a way that comments probably won't